core/server/models/session.js

```diff
@@ -0,0 +1,54 @@
+const ghostBookshelf = require('./base');
+
+let Session,
+    Sessions;
+
+Session = ghostBookshelf.Model.extend({
+    tableName: 'sessions',
+
+    parse: function parse(attrs) {
+        if (!attrs.session_data) {
+            return attrs;
+        }
+        return Object.assign(attrs, {
+            session_data: JSON.parse(attrs.session_data)
+        });
+    },
+
+    format: function format(attrs) {
```

Comment:
+1 for using `format` and `parse` 👍

Reply:
out of interest, what else might i have done?

Reply:
You could have parsed the session data in the CRUD functions.